src/modules/universal-discovery/components/content-table/content.table.item.component.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './css/content.table.item.component.css';
+
+const ContentTableItemComponent = (props) => {
+    const { onItemClick, onPreview, data, contentTypesMap, labels } = props;
+
+    const item = data.ContentInfo.Content;
+    const contentType = contentTypesMap ? contentTypesMap[item.ContentType._href] : false;
+    const contentTypeName = contentType ? contentType.names.value[0]['#text'] : labels.notAvailable;
+    const onClick = !!onItemClick ? onItemClick.bind(null, data) : null;
```

Comment:
Why do we need this check? I think this props is required?

Reply:
I removed `isRequired` flag form `propTypes`.